aws-ts-ruby-on-rails/ami.ts

```diff
@@ -0,0 +1,95 @@
+import * as aws from "@pulumi/aws";
+
+const instanceTypeToArch: {[instanceType: string]: string} = {
+    "t1.micro"    : "PV64" ,
```

Comment:
FWIW- this list is missing about half the EC2 instance types.

Reply:
Eek! I took this straight from the most recent Application Template in the AWS Console!

I'd happily replace this with a call to some standard implementation of this. Should we just create a simple unofficial package for this? Or, do we prefer using the dynamic querying variant that I blogged about recently.

I was trying to keep this as faithful a port as possible, and am not keen to get into the business of manually maintaining each and every list here, but I agree it's a shame if people copy this and it doesn't work for all possible regions.